### ISSUE TITLE
Failed to parse build gradle error:  failed to parse compileSDKVersion ()

### ISSUE THREAD

**Issue by mauker** [bitrise-steplib/steps-install-missing-android-tools#24]
I don't have this problem in Android Studio, this problem appeared today, but I try using the release created with successful yesterday doesn't run now with a simple rebuild, can you help us please:

Failed to parse build gradle file: /bitrise/src/app/build.gradle, error: failed to determine compileSDKVersion, error: failed to find compileSdkVersion
Switched to extended analyzer...
Failed to analyze build.gradle at: /bitrise/src/app/build.gradle, error: failed to parse compileSDKVersion ()

[d49091c3144960f1.log](https://github.com/bitrise-steplib/steps-install-missing-android-tools/files/1377422/d49091c3144960f1.log)

**Comment by koral--** [comment]
```
Caused by: java.lang.RuntimeException: You have not accepted the license agreements of the following SDK components:
[Android SDK Build-Tools 25].
Before building your project, you need to accept the license agreements and complete the installation of the missing components using the Android Studio SDK Manager.
Alternatively, to learn how to transfer the license agreements from one workstation to another, go to http://d.android.com/r/studio-ui/export-licenses.html
```
That's the source of the issue.
You are using quite old version: `install-missing-android-tools@1.0.3`
It was fixed in 2.0.3: https://github.com/bitrise-steplib/steps-install-missing-android-tools/releases/tag/2.0.3

**Comment by viktorbenei** [comment]
Indeed, thanks @koral-- , you're absolutely right!

@mauker please upgrade the step to the latest version, that should help! :)

I'll close this issue, but if it wouldn't work even after the step version upgrade just let us know and we'll reopen it! :)
